### PR TITLE
Update to elastic/beats@cf959bfb535e, Go 1.14.7

### PR DESCRIPTION
**Pull request by axw**
## Motivation/summary

Update libbeat to fix release manager, and update Go version to match Beats.

## Checklist

- [x] I have signed the [Contributor License Agreement](https://www.elastic.co/contributor-agreement/).
- [x] I have updated [CHANGELOG.asciidoc](https://github.com/elastic/apm-server/blob/master/CHANGELOG.asciidoc)

I have considered changes for:
~- [ ] documentation~
~- [ ] logging (add log lines, choose appropriate log selector, etc.)~
~- [ ] metrics and monitoring (create issue for Kibana team to add metrics to visualizations, e.g. [Kibana#44001](https://github.com/elastic/kibana/issues/44001))~
- [x] automated tests (add tests for the code changes, all [**unit** tests](https://github.com/elastic/apm-server/blob/master/TESTING.md) pass locally)
~- [ ] telemetry~
~- [ ] Elasticsearch Service (https://cloud.elastic.co)~
~- [ ] Elastic Cloud Enterprise (https://www.elastic.co/products/ece)~
~- [ ] Elastic Cloud on Kubernetes (https://www.elastic.co/elastic-cloud-kubernetes)~

## How to test these changes

`make test release`

## Related issues

None.

### CODE REVIEW

File: changelogs/head.asciidoc [hunk 1]
```diff
@@ -22,3 +22,4 @@ https://github.com/elastic/apm-server/compare/7.9\...master[View commits]
 * Set destination.service.* from http.url for Jaeger spans {pull}4046[4046]
 * Use service.version for Metadata.Service.Version when converting a Jaeger span {pull}4061[4061]
 * Add transaction.experience fields {pull}4056[4056]
+* Upgrade Go to 1.14.9 {pull}4067[4067]
```
Comment (jalvz) [inline changelogs/head.asciidoc:25]: 1.14.7?

Comment (axw) [reply changelogs/head.asciidoc:25]: :facepalm:

Comment (axw) [inline changelogs/head.asciidoc:25]:
```suggestion
* Upgrade Go to 1.14.7 {pull}4067[4067]
```

Comment (axw) [reply changelogs/head.asciidoc]: *And* I had 1.14.4 in the title

**Pull request merged**